client/src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Icons from '../Icons/FAIcon';
+import './NavBar.css';
+import logo from './Logo/Logo.png';
+
+const NavBar = () => (
+    <div>
+        <nav className='navBar'>
+            <div>
+                <Icons name={['fas', 'bars']}/>
```

Comment:
This should be the Hamburger icon that Alex is working on

Reply:
Did you miss this one too?

Reply:
Added a placeholder until the Hamburger Component is finished